README.md

```diff
@@ -1,161 +1,187 @@
 
 (amongst others) which calls the specified zome function, blocks, and returns the result.
 
 This enables us to write code like this:
 
 ```javascript
-let dnaHash = Container.installedApps()[0]
-let app = Container.instantiate(dnaHash)
+var dnaHash = Container.installedApps()[0]
+var app = Container.instantiate(dnaHash)
 // app is actually a C++ object that wraps the Rust Holochain instance
 app.name() // -> App name defined in DNA
 app.start() // starts the instance's event loop and later will start the network node and DHT (not implemented in Rust yet)
-let result = app.call("zome name", "capability name", "function name", "parameters") // runs the zome function which invovles interpreting the app's WASM code
+var result = app.call("zome name", "capability name", "function name", "parameters") // runs the zome function which involves interpreting the app's WASM code
 ```
 
+To see fuller documentation on this, please visit the [example JS tests repo](https://github.com/holochain/js-tests-scaffold).
+
+## Qt <> C Bindings for holochain-rust
+
+The **bindings** folder contains the Qt/C++ wrappers around the Holochain Rust code that is made available in the form of plain C functions and structs. This project makes Holochain available as Qt-based QObjects that can easily be plugged into Qt script engines. This project compiles to a static library.
+
+# Installation
 
-### Using Tape testing framework
-To get proper assertions and formatted output we want to use existing JS scripting frameworks. We chose Tape for now because of its mininmal footprint.
+While this repository is still pre-release, there are no pre-built libraries yet, so the source code must be cloned and compiled on your computer. In the future, installation of these tools will be dramatically simplified.
 
-In order to build a JS bundle including Tape, we use Webpack. `app-test-scaffold` contains a working example which can be run like this:
+### Dev Dependencies
+
+* [Rust](https://www.rust-lang.org/en-US/install.html)
+	* The `nightly` toolchain of Rust is needed for compiling Holochain, `stable` will not work
+	* Once you follow the initial install instructions, run the following, which install the nightly toolchain, and set it as the default toolchain:
+	* `rustup toolchain install nightly`
+	* `rustup default nightly`
+* [Qt](http://doc.qt.io/qt-5/index.html)
+	* Qt is a framework for C++, which is used to interface with the C bindings of holochain-rust
+	* For macOS (and with [brew](https://brew.sh/) on your computer), to install just run: `brew install qt`
+	* For linux systems you may need to install [some dependencies](http://doc.qt.io/qt-5/linux.html) for successful build
+* [cmake](https://cmake.org/install)
+	* `cmake` is required in your CLI to accomplish the `make` compile steps that follow
+
+ First, clone this repo to your computer. Do not name your folder anything different than `holosqape`, you will break the build.
```

Comment:
Why is that?!

Reply:
Because there are hardcoded path references to `holosqape` folder in the files